Sources/Controllers/Map/Widgets/OATopTextView.mm

```diff
@@ -497,7 +497,7 @@ - (BOOL) updateInfo
         {
             if ([_settings.showStreetName get])
             {
-                OANextDirectionInfo *nextDirInfo = [_routingHelper getNextRouteDirectionInfo:_calc1 toSpeak:YES];
+                OANextDirectionInfo *nextDirInfo = [_routingHelper getNextRouteDirectionInfo:[[OANextDirectionInfo alloc] init] toSpeak:YES];
```

Comment:
Why _calc1 replaced to [[OANextDirectionInfo alloc] init]?

Reply:
https://github.com/osmandapp/OsmAnd/blob/44259765c7829cf199f945d90d9583190bab5bd6/OsmAnd/src/net/osmand/plus/views/mapwidgets/widgets/StreetNameWidget.java#L427